src/test/java/uk/ac/cam/cl/dtg/isaac/api/GroupsFacadeIT.java

```diff
@@ -43,23 +43,23 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.easymock.EasyMock.*;
+import static org.easymock.EasyMock.replay;
 import static org.junit.Assert.assertEquals;
 import static uk.ac.cam.cl.dtg.isaac.api.ITConstants.*;
 
 
-public class GroupsFacadeIT extends IsaacIntegrationTest {
+public class GroupsFacadeIT extends IsaacIntegrationJupiterTest {
 
     private GroupsFacade groupsFacade;
 
-    @Before
+    @BeforeEach
```

Comment:
How come this was changed? I'm not saying it's wrong, but why was this done?

Reply:
Before and BeforeEach are the Junit4 and 5 versions respectively of the same annotation. Parameterised tests weren't playing nice with the Junit4 version and mixing the Before versions sometimes caused issues as well. Since the parameterised apparently needed me to update the base IT class, I updated the other tests that extended on it to prevent any other versioning issues.

Reply:
Cool, that makes sense, thanks for explaining!